x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx

```diff
@@ -326,27 +326,38 @@ export const ResponseActionsLog = memo<
         name: TABLE_COLUMN_NAMES.hosts,
         width: '20%',
         truncateText: true,
-        render: (hosts: ActionDetails['hosts']) => {
+        render: (_hosts: ActionDetails['hosts']) => {
+          const hosts = _hosts && Object.values(_hosts);
           // join hostnames if the action is for multiple agents
           // and skip empty strings for names if any
-          const hostname =
-            hosts &&
-            Object.values(hosts)
-              .reduce<string[]>((acc, host) => {
-                if (host.name.trim()) {
-                  acc.push(host.name);
-                }
-                return acc;
-              }, [])
-              .join(', ');
+          const _hostnames = hosts
```

Comment:
This render function is a little hard to read, but I think you're missing a condition for when the action was sent to multiple agents, but only a subset of those are now un-enrolled. 

Example: Action sent to:

- `agent-a`
- `agent-b`
- `agent-c`

`agent-b` has since been unenrolled and we are not able to get its host name. 

This case, the display (I think) should be something like:

```
agent-a-host-name, Host unenrolled, agent-c-host-name
```

Also, please adjust the comment on line 331-332 to be reflective of this new code

Reply:
I see what you mean. I went with the logic to not show the unenrolled host when one or several are missing of many agents, in order to avoid seeing multiple `Host unenrolled` in between names (if such a case arises).

What you're suggestions might look like `Name a, Host unenrolled, Host unenrolled, Host unenrolled, Host unenrolled...` or worst `Host unenrolled, Host unenrolled...` times X, until there's one name that's truncated and the only way to see that would be to see it on hover tooltip. 

Alternatively, we can group all the host names in the beginning of the list so they are seen first and then the unenrolled names. `Name 1, Name 50, Name 254, Host unenrolled, Host unenrolled...` etc. That way at least the existing host names would be seen right away without hovering on the tooltip. @paul-tavares

Reply:
Ok. I'm ok leaving this as you have it.